fluentbit/conf/conf_test.go

```diff
@@ -798,12 +850,155 @@ func TestGenerateFluentBitMainConfigErrors(t *testing.T) {
 		},
 	}
 	for _, tc := range tests {
-		if _, err := GenerateFluentBitMainConfig(tc.tails, tc.syslogs, nil, nil, nil, nil, nil); err == nil {
+		if _, err := GenerateFluentBitMainConfig(tc.tails, tc.syslogs, nil, nil, nil, nil, nil, nil); err == nil {
 			t.Errorf("test %q: GenerateFluentBitMainConfig succeeded, want error", tc.name)
 		}
 	}
 }
 
+func TestGenerateFluentBitMainConfigWindows(t *testing.T) {
+	tests := []struct {
+		name    string
+		tails   []*Tail
+		winlogs []*WindowsEventlog
+		want    string
+	}{
+		{
+			name: "multiple tail and winlog plugins",
+			tails: []*Tail{{
+				Tag:  "test_tag1",
+				DB:   "test_db1",
+				Path: "test_path1",
```

Comment:
Curious: Was there any special logic we need to adjust to make Windows style file path work? e.g. `C:\Program Files...`

Reply:
I don't think it has some special logic. any `/var/log` will go under C:\ i think. For example, for google-fluentd, we have this `buffer_path /var/log/google-fluentd/buffers` https://github.com/GoogleCloudPlatform/google-fluentd/blob/master/windows-installer/fluent-template.conf#L32 and it just creates in `C:\var\log\google-fluentd\buffers`

Reply:
It's different template engine so you might need to confirm first.

It's possible that you might need to use `filepath.Dir` to render the path correctly on Windows, including the code for other paths.

Reply:
I verified that in_tail plugin and in_winlog plugin running on Windows understand file paths  "/var/log/file1" and "C:\var\log\file1". Therefore, I think we are good with the current code. The only diff i noticed is that when tailing multiple files, for  syntax - "/var/log_dir/file1", "/var/log_dir/file2", you have to list out all file name one by one. To allow wildcard, you can only use "C:\var\log_dir\*".